res/css/views/rooms/_EventTile.scss

```diff
@@ -27,37 +27,6 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
 
     flex-shrink: 0;
 
-    &.mx_EventTile_highlight,
-    &.mx_EventTile_highlight .markdown-body {
-        color: $alert;
-    }
-
-    &.mx_EventTile_bubbleContainer {
-        display: grid;
-        grid-template-columns: 1fr 100px;
-
-        .mx_EventTile_line {
-            margin-right: 0;
-            grid-column: 1 / 3;
-            padding: 0 !important; // override default padding of mx_EventTile_line so that we can be centered
-        }
-
-        .mx_EventTile_msgOption {
-            grid-column: 2;
-        }
-
-        &:hover {
-            .mx_EventTile_line {
-                // To avoid bubble events being highlighted
-                background-color: inherit !important;
-            }
-        }
-    }
-
-    &.mx_EventTile_isEditing .mx_MessageTimestamp {
-        visibility: hidden;
-    }
-
```

Comment:
Please note the rules above were moved from the block with `:not()` pseudo class with recent PRs.